spec/models/generic_announcement_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe GenericAnnouncement, type: :model do
+  describe '.default_scope' do
```

Comment:
You aren't really describing `.default_scope`, I'd just put this in the `describe` block.